### PR TITLE
bootstrap.sh: use the real version number instead of hardcoding "current"

### LINKED ISSUE #126: Update to new version
Is there a way to update to Toolbox 1.0.4 ? 

curl -sL toolbox.qutheory.io | bash and brew install vapor/tap/toolbox still installs 1.0.3

### PR DESCRIPTION
**Pull request by vzsg**
This PR enhances the installer script to overwrite the hardcoded version in `Executable/main.swift` with the current Git tag description, i.e. the actual version number.

This resolves #126 and prevents any future confusion about the Toolbox. See the extended description in the commits for some extra explanation.

### CODE REVIEW

File: Sources/Executable/main.swift [hunk 1]
```diff
@@ -3,8 +3,7 @@ import Console
 import Foundation
 import VaporToolbox
 
-let version = "1.0.3"
-
+let version = "master"
```
Comment (tannernelson) [inline Sources/Executable/main.swift]: Can you add a comment here that mentions not to modify this line at all?

Comment (vzsg) [inline Sources/Executable/main.swift]: Sure.

Comment (tannernelson) [inline Sources/Executable/main.swift]: Once done I can merge and tag this with the shallow clone fix as well.

**Comment by vzsg** [comment]
Sorry about the delay, I added the comment just now.

**Pull request merged**